requirements/app/test.txt

```diff
@@ -8,6 +8,6 @@ httpx
 trio<0.22.0
 pympler
 psutil
-setuptools<=59.5.0
+setuptools<65.6.0
```

Comment:
I recall there was a compatibility reason, but it was maybe fixed in later versions...